pkg/config/setup/config.go

```diff
@@ -1061,6 +1061,8 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("admission_controller.container_registry", "gcr.io/datadoghq")
 	config.BindEnvAndSetDefault("admission_controller.timeout_seconds", 10) // in seconds (see kubernetes/kubernetes#71508)
 	config.BindEnvAndSetDefault("admission_controller.service_name", "datadog-admission-controller")
+	config.BindEnvAndSetDefault("admission_controller.autoscaling.enabled", false)
+	config.BindEnvAndSetDefault("admission_controller.autoscaling.endpoint", "/autoscale")
```

Comment:
I don't think we need a parameter to define the endpoint path

Reply:
We have one for every other webhook